pkg/controller/cassandracluster/reconcile.go

```diff
@@ -530,7 +529,12 @@ func UpdateCassandraClusterStatusPhase(cc *api.CassandraCluster, status *api.Cas
 
 			rackName := cc.GetRackName(dc, rack)
 			dcRackName := cc.GetDCRackName(dcName, rackName)
-			dcRackStatus := status.CassandraRackStatus[dcRackName]
+			dcRackStatus, ok := status.CassandraRackStatus[dcRackName]
```

Comment:
You could rename ok and use something like `dcRackExists` which makes more sense

Reply:
have changed it to `exist`